UefiPayloadPkg/Library/FdtParserLib/FdtParserLib.c

```diff
@@ -880,6 +925,8 @@ ParseDtb (
         DEBUG ((DEBUG_INFO, "After ParsePciRootBridge, index :%x\n", index));
         break;
       case Options:
+        // FIXME: Need to ensure this node gets parsed first so that it gets

```

Comment:
good catch.
I think we can handle cross dependency outside the node parsing loop.
For example, move below line from ParsePcieRootBridge () to outside the for...loop so all the nodes already parsed:
    mPciRootBridgeInfo->ResourceAssigned = (BOOLEAN)PciEnumDone;

Reply:
I have done it that way but I do not think this is a good long term solution. We need to discuss this in the follow up sessions and we should avoid such cross node dependencies. Maybe RB should have its own field for this.

Reply:
resolved